docs/reference/how-to/fix-common-cluster-issues.asciidoc

```diff
@@ -1,7 +1,104 @@
 [[fix-common-cluster-issues]]
 == Fix common cluster issues
 
-This guide describes how to fix common problems with {es} clusters.
+This guide describes how to fix common errors and problems with {es} clusters.
+
+[discrete]
+=== Error: disk usage exceeded flood-stage watermark, index has read-only-allow-delete block
+
+This error indicates a data node is critically low on disk space and has reached
+the <<cluster-routing-flood-stage,flood-stage disk usage watermark>>. When a
+node reaches this watermark, {es} blocks writes to any index with a shard on the
+node to prevent a full disk. If the block affects related system indices, {kib}
+and other {stack} features may become unavailable.
+
+{es} will automatically remove the write block when the affected node goes below
+the <<cluster-routing-watermark-high,high disk watermark>>. To achieve this,
+{es} automatically moves some of the affected node's shards to other nodes in
+the data tier. However, {es} will only allocate shards to nodes below the high
```

Comment:
s/high/low/ here.

```suggestion
the data tier. However, {es} will only allocate shards to nodes whose disk usage is below the low
```

Also there may be other reasons it can't allocate shards to other nodes, e.g. they're in the wrong data tier or there's an allocation filter, or allocation awareness prevents it, or ...

Reply:
Good point and thanks for catching my error re: the high/low watermarks. I think just pointing folks to the cat allocation API will suffice here.